.github/workflows/runtests.yml

```diff
@@ -172,11 +172,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.9', '3.10', 'pypy-3.8']
+        python-version: ['3.9', '3.10', '3.11', 'pypy-3.8']
         experimental: [false]
-        include:
-          - python-version: '3.11.0-alpha - 3.11'
-            experimental: true
+        # Maybe use this to add 3.12 when the time comes:
+        #include:
+        #  - python-version: '3.12.0-alpha - 3.12'
+        #    experimental: true
```

Comment:
Can this not be done now?

Reply:
It doesn't look like it is possible to install?

Reply:
https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md

Reply:
PyPy
2.7.18 [PyPy 7.3.9]
3.6.12 [PyPy 7.3.3]
3.7.13 [PyPy 7.3.9]
3.8.13 [PyPy 7.3.9]
3.9.12 [PyPy 7.3.9]
Python
2.7.18
3.6.15
3.7.15
3.8.14
3.9.15
3.10.8
3.11.0

Reply:
Not sure where the required tests is set, but it should be changed....

```
doctests (3.11.0-alpha - 3.11, false) Expected — Waiting for status to be reported
Required
tests1 (3.11.0-alpha - 3.11, false) Expected — Waiting for status to be reported
Required
tests2 (3.11.0-alpha - 3.11, false) Expected — Waiting for status to be reported
```

Reply:
Maybe this is holding it up: https://github.com/actions/python-versions/pull/197

Reply:
> Not sure where the required tests is set, but it should be changed....

It's in the repo settings in the branch protection rules. I've just removed those required jobs. New jobs will have to be set as required when the list is finalised.